src/components/PatientRecord/PatientRecord.module.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../styles/variables';
+
+// ----------  TITLE SPACING  --------- //
+$verticalPadding: 20px;
+$lineHeight: 30px;
+
+.record {
+    display: flex;
+    color: $white;
+
+    .recordToggle {
+        display: inline-flex;
+        float: left;
+        width: 65px;
+        height: 65px;
+        font-size: 2.5em;
+        align-items: center;
+        justify-content: center;
+        background-color: $blue;
+
+        &:hover {
+            cursor: pointer;
+        }
+    }
+
+    .sidebar {
+        float: left;
+        width: 650px;
+        padding: 25px;
+        background-color: $gray;
+        overflow-y: scroll;
+    }
+
+    .element {
+        display: block;
+        margin-bottom: 1vh;
+        font-size: 1.1em;
+        text-transform: uppercase;
```

Comment:
This seems to be transforming all text, not just the headers. Per the design I think we want the internal content to retain its original case.
<img width="611" alt="image" src="https://user-images.githubusercontent.com/13512036/78137887-09b08800-73f4-11ea-9ac4-f482927deb3b.png">

Also this is very subjective but the all-caps section headers look a little odd to me.
<img width="605" alt="image" src="https://user-images.githubusercontent.com/13512036/78138049-51371400-73f4-11ea-8e89-2f62e10696ce.png">


 I can't quite put my finger on what specifically the difference between the design and implementation is but somehow the design looks "friendlier"

Reply:
Updated text-transform to be on the title only. Increased the font size and added a small letter spacing to make it look friendlier
![Screen Shot 2020-04-01 at 8 57 31 AM](https://user-images.githubusercontent.com/53485517/78139650-d6232d00-73f6-11ea-80e3-181834de5c36.png)